backend/test/api/base_test.clj

```diff
@@ -38,3 +38,37 @@
 (defexpect join-empty
   (expect {:error messages/lobby-not-created}
               (base/add-player 0)))
+
+(defexpect play-card-messages
+
+  (let [game (base/create-game)
+        game-id (:game-id game)
+        p1 (:player-id game)
+        p2 (:player-id (base/add-player game-id))]
+    
+    (expect {:error messages/invalid-id}
+            (base/play-card-as-player 9999999999 p1 0 0))
```

Comment:
@kenan-rhoton apparently "mocking" is making this test not as good as it should be